gen/output_template.tmpl

```diff
@@ -1,6 +1,9 @@
+{{- if len .PackageDoc }}
+// {{ .PackageDoc }}
+{{- end }}
 package {{ .PackageName }}
 
-// Code generated by avro/gen. DO NOT EDIT.
+// Code generated by avro/gen. DO NOT EDIT MANUALLY.
```

Comment:
It is conventional for this to be `DO NOT EDIT.`. I also think (no proof) that some tools look for this specific string.